src/app/public/modules/summary-action-bar/actions/summary-action-bar-cancel.component.ts

```diff
@@ -14,9 +14,15 @@ import {
 })
 export class SkySummaryActionBarCancelComponent {
 
+/**
+ * Indicates whether to disable the cancel action.
+ */
   @Input()
   public disabled = false;
 
+/**
+ * Fires when users click the cancel action.
```

Comment:
I think `click` should be `select` in all three event descriptions. I'm pretty sure the old docs pre-date the SCD terminology rule (or at least pre-date it filtering down).